dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -19,7 +19,7 @@ CommandBarFlyoutCommandBar::CommandBarFlyoutCommandBar()
     Loaded({
         [this](auto const&, auto const&)
         {
-            COMMANDBARFLYOUT_TRACE_INFO(*this, TRACE_MSG_METH, METH_NAME, this);
+            //COMMANDBARFLYOUT_TRACE_INFO(*this, TRACE_MSG_METH_STR, METH_NAME, this, L"Loaded");
```

Comment:
You've left a lot of these commented out.  Is there a reason not to just delete them?

Reply:
Yes, see my comment in my opening description. I did not want to add more literals to the dll, and at the same time I want to easily re-enable them in future debugging.

Reply:
Ah, gotcha, so you're wanting them there so you can uncomment them if need be.  That makes sense.

Reply:
A Preprocessor directive is probably better than commented out code for this.